pr_check.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
```

Comment:
"make test" could go here perhaps, now or in future. (just the unit tests) Not sure how worthwhile it is though as we would have had to pass that and several other checks to get to master.

Reply:
yeah, I'd say for the moment keep this exit 0, as you already have a testing process

Reply:
actually perhaps doing `make docker-build` here would also be interesting, are you testing this with `BUILD_CMD="docker build"` elsewhere? this would ensure that it builds correctly at least. Do you agree? (just build, no push)

Reply:
I think that would be a good check, that may not be used at all in openshift CI.